src/components/editor/rx/share.ts

```diff
@@ -64,22 +60,23 @@ const makeDefaultWorkspace = () =>
 
 export const importRx = runtime
   .rx((get) =>
-    Effect.gen(function* () {
+    Effect.gen(function* (_) {
       const hash = get(hashRx)
       if (hash._tag === "None") return makeDefaultWorkspace()
       const compressed = yield* Effect.promise(() =>
         retrieveCompressed(hash.value)
       )
       if (!compressed) return makeDefaultWorkspace()
       const compression = yield* WorkspaceCompression
-      return yield* compression
-        .decompress({
+      return yield* _(
```

Comment:
Not that it really matters, but we can probably start removing the adapter usage when we see it.

Reply:
I added it on purpose lol, sometimes the .pipe() method formats weird. Can replace with normal pipe.

Reply:
I figured it was probably on purpose lol